quickjspp.hpp

```diff
@@ -498,7 +500,7 @@ struct unwrap_arg_impl {
     {
         if (size_t(argc) <= I) {
             JS_ThrowTypeError(ctx, "Expected type %lu arguments but only %d were provided",
-                              (unsigned long)NArgs, argc);
+                              NArgs, argc);
```

Comment:
https://github.com/ftk/quickjspp/runs/4259581835?check_suite_focus=true#step:5:89

Reply:
Missed that. Should be fixed now.